### ISSUE TITLE
Error in pyproject.toml:  {...} is not valid under any of the given schemas

### ISSUE THREAD

**Issue by iliya-malecki** [tamasfe/taplo#368]
[project] section with anything in it causes the following problem:

```
[project]
```
> {} is not valid under any of the given schemas

```
[project]
name = "hello"
```
> {"name":"hello"} is not valid under any of the given schemas

**Comment by stdedos** [comment]
Additionally, it seems that

```json
    "evenBetterToml.schema.associations": {
        "pyproject.toml": "https://json.schemastore.org/pyproject.json",
    },
```

Does not force detection either

**Comment by hongquan** [comment]
It is because you missed "version" key: https://github.com/tamasfe/taplo/issues/316

**Comment by iliya-malecki** [comment]
@tamasfe should this one be closed as a duplicate? we have #316 where the path seems clear to me: patching. Please comment on what you think is necessary with the situation

**Comment by ia0** [comment]
Yes this looks like a duplicate. I'll take a look at #316 when I'll have access to a computer in a few weeks. Thanks for helping triaging the issues!
